### PR TITLE
theme color

### PR DESCRIPTION
**Pull request by GoodThomas**

### CODE REVIEW

**Comment by GoodThomas** [comment]
Button and form need new font constants, but they are not determined.

File: src/config/theme.js [hunk 1]
```diff
@@ -24,34 +24,46 @@ const lineHeightLg = '133.33%';
...
+const textColorDarkGrey = '#333333';
+const textColorMediumGrey = '#666666';
+const textColorLightGrey = '#b3b3b3';
 
-// Red
-const redColor = '#FE4A49';
-const redColorLight = '#FFDEDE';
-const redColorDark = '#960F0E';
+// Background Color
+const backgroundColorLight = '#f9f9f9';
```
Comment (ippudo) [inline src/config/theme.js]: Will we have other darker background colors?

Comment (GoodThomas) [reply src/config/theme.js]: Sorry, I don't know. But now there is only one background color.
image

File: src/config/theme.js [hunk 2]
```diff
@@ -24,34 +24,46 @@ const lineHeightLg = '133.33%';
...
 
-// Orange
-const orange = '#F5A623';
+// Support Colors
+const supportColorRed100 = '#d0021b';
```
Comment (dwalintukan) [inline src/config/theme.js]: BE2828

File: src/config/theme.js [hunk 3]
```diff
@@ -24,34 +24,46 @@ const lineHeightLg = '133.33%';
...
+const supportColorRed60 = '#e36776';
```
Comment (dwalintukan) [inline src/config/theme.js]: D36E78

File: src/config/theme.js [hunk 4]
```diff
@@ -24,34 +24,46 @@ const lineHeightLg = '133.33%';
...
+const supportColorRed30 = '#f1b3bb';
```
Comment (dwalintukan) [inline src/config/theme.js]: E8B6BC

File: src/config/theme.js [hunk 5]
```diff
@@ -24,34 +24,46 @@ const lineHeightLg = '133.33%';
...
+const supportColorBlue100 = '#1e81dc';
```
Comment (dwalintukan) [inline src/config/theme.js]: 3F7FD5

File: src/config/theme.js [hunk 6]
```diff
@@ -24,34 +24,46 @@ const lineHeightLg = '133.33%';
...
+const supportColorBlue60 = '#78b3ea';
```
Comment (dwalintukan) [inline src/config/theme.js]: 85B2E5

File: src/config/theme.js [hunk 7]
```diff
@@ -24,34 +24,46 @@ const lineHeightLg = '133.33%';
...
+const supportColorBlue30 = '#bbd9f5';
```
Comment (dwalintukan) [inline src/config/theme.js]: C1D8F2

File: src/config/theme.js [hunk 8]
```diff
@@ -24,34 +24,46 @@ const lineHeightLg = '133.33%';
...
+const supportColorGreen100 = '#64c850';
```
Comment (dwalintukan) [inline src/config/theme.js]: 7EC660

File: src/config/theme.js [hunk 9]
```diff
@@ -24,34 +24,46 @@ const lineHeightLg = '133.33%';
...
+const supportColorGreen60 = '#a2de96';
```
Comment (dwalintukan) [inline src/config/theme.js]: AFDC9D

File: src/config/theme.js [hunk 10]
```diff
@@ -24,34 +24,46 @@ const lineHeightLg = '133.33%';
...
+const supportColorGreen30 = '#d1efcb';
```
Comment (dwalintukan) [inline src/config/theme.js]: D7EECE

File: src/config/theme.js [hunk 11]
```diff
@@ -24,34 +24,46 @@ const lineHeightLg = '133.33%';
...
+const supportColorYellow100 = '#f5a623';
```
Comment (dwalintukan) [inline src/config/theme.js]: E9AA44

File: src/config/theme.js [hunk 12]
```diff
@@ -24,34 +24,46 @@ const lineHeightLg = '133.33%';
...
+const supportColorYellow60 = '#f9ca7b';
```
Comment (dwalintukan) [inline src/config/theme.js]: F1CB86

File: src/config/theme.js [hunk 13]
```diff
@@ -24,34 +24,46 @@ const lineHeightLg = '133.33%';
...
+const supportColorYellow30 = '#fce4bd';
```
Comment (dwalintukan) [inline src/config/theme.js]: F8E5C1

**Pull request merged**